baas-smoke-tests/src/test/scala/com/ing/baker/baas/smoke/BakeryFunSpec.scala

```diff
@@ -21,6 +21,16 @@ abstract class BakeryFunSpec extends fixture.AsyncFunSpecLike {
   implicit val timer: Timer[IO] =
     IO.timer(executionContext)
 
+  val isWindows: Boolean = System.getProperty("os.name").toLowerCase.contains("windows")
```

Comment:
Please use something like this:

`sys.props.get("os.name").forall(_.toLowerCase().contains("windows"))`

Reply:
Should be this field be a part of getPathSafe method? For now it's a public field